src/helperFunctions.js

```diff
@@ -7,15 +7,24 @@ function writeToFile(filePath, txt) {
   return fs.writeFileSync(filePath, body);
 }
 
+function hasApiKey(api_key) {
+  if(api_key !== '' || typeof(api_key) !== 'undefined') {
+    return false
+  } else {
+    return true
+  }
```

Comment:
The boolean logic is kind of hard to follow here. Might it be better to use `===` in the if check and adjust the boolean operator and/or true and false cases accordingly?

For example, it seems like right now it will return false when the api key is not blank (short-circuited || operator), but that seems wrong.